types/office-js-preview/index.d.ts

```diff
@@ -125893,7 +125893,7 @@ declare namespace PowerPoint {
     enum ErrorCodes {
         generalException = "GeneralException",
     }
-    namespace Interfaces {
+    module Interfaces {
```

Comment:
IIRC, we were asked to change this from `module` to `namespace`. All the other instances in our files say `namespace`.

```suggestion
    namespace Interfaces {
```